Library/Config.ts

```diff
@@ -13,6 +15,9 @@ class Config {
     public static ENV_profileQueryEndpoint = "APPINSIGHTS_PROFILE_QUERY_ENDPOINT";
     public static ENV_quickPulseHost = "APPINSIGHTS_QUICKPULSE_HOST";
 
+    // Azure Connection String
+    public static ENV_connectionString = "APPINSIGHTS_CONNECTIONSTRING";
```

Comment:
This is the wrong environment variable. Please use "APPLICATIONINSIGHTS_CONNECTION_STRING"

Reply:
Resolved in 91297e5